Development v1.0

kumaraswamy1 opened the pull request:
Raising PR for reviews and feedback.

live site: [pigment-UI](https://pigment.netlify.app/)

 Components implemented
- Avatar
- Alert
- Badge
    - badge on icons
    - badge on avatars
- Button
    - primary
    - secondary
    - pills
- Card
    - text only cards
    - image card
    - social media card
    - ecommerce card

- Image
    - responsive image
    - round image
    - square image
- Input
    - textbox
    - radio button
    - checkbox
- Lists
    - stacked list
    - nested list
- Navigation
    - desktop (Simple)
    - navbar with hamburger
- Modal
- Rating (for ecommerce)
- Simplified Grid
    - grid-2: two items in a grid
    - grid-3: three items
- Text Utilities
    - headings
    - small text
    - gray text
    - center text
 - Toast

kumaraswamy1 committed:
feat : Adding Components

kumaraswamy1 committed:
fix: image format

kumaraswamy1 committed:
style: fixing overflow

kumaraswamy1 committed:
fix : folder structure

kumaraswamy1 committed:
fix: Changed clipboard id

codingwolf-at commented on Components/Navigation/navigation.html:
```diff
@@ -0,0 +1,172 @@
+        <h2>Simple Navbar</2>
+          <div class="nav">
+            <div class="nav-list default">
+              <li><a href="http://">Home</a></li>
+              <li><a href="http://">Products</a></li>
+              <li><a href="http://">About</a></li>
+              <li><a href="http://">Blog</a></li>
```
Clicking on the link user is being directed to a blank page.
Instead of using `https://` you can use `javascript:void(0);` this will make it a dead link.

kumaraswamy1 replied:
Thanks. Made all the changes.

codingwolf-at commented on index.html:
```diff
@@ -0,0 +1,80 @@
+      <p> 2021 | Inspired by <span><a href="https://getbootstrap.com/"> Bootstrap</a></span></p>
+    </div>
+    <div class="center social-links">
+      <a href="https://twitter.com/thekumaraswamy"><i class="fab fa-twitter"></i></a>
+      <a href="https://www.linkedin.com/in/kumaraswamya/"><i class="fab fa-linkedin-in"></i></a>
+      <a href="https://www.instagram.com/thekumaraswamy/"><i class="fab fa-instagram"></i></a>
```
Maybe add `target="_blank"` attribute to your social links, so that user won't leave your doc site after clicking on your socials.

codingwolf-at commented on index.html:
```diff
@@ -0,0 +1,80 @@
+          </span>
+        </button>
+        <button class="btn-secondary">
+          <a href="https://github.com/kumaraswamy1/Component-library">
```
update the GitHub url, this is giving a 404 page 
and you can also give `target="_blank"` here

kumaraswamy1 replied:
Done

codingwolf-at reviewed (commented):
Nice work there! 
The code also looks clean to me, loved the copy code snippet button.

kumaraswamy1 committed:
docs: updating CSS link

kumaraswamy1 committed:
docs: Adding Component.js file

kumaraswamy1 committed:
Update docs.html

kumaraswamy1 committed:
refactor: Added JS files

kumaraswamy1 committed:
docs: updated Github link

kumaraswamy1 committed:
refactor: Added target in link tags

kumaraswamy1 committed:
docs: added stylesheet link

kumaraswamy1 committed:
refactor : Fixed margin

kumaraswamy1 committed:
refactor : adding all files to Component.js

kumaraswamy1 committed:
refactor : Adding js files

kumaraswamy1 committed:
style: fixing badge height

kumaraswamy1 committed:
style : change in primary color

kumaraswamy1 committed:
feat: added single js file for all the components

kumaraswamy1 committed:
feat: Changed the components to single html file

kumaraswamy1 committed:
refactor: Changed folder structure

kumaraswamy1 committed:
style: changes to component

kumaraswamy1 committed:
style: removed custom scrollbar

kumaraswamy1 committed:
refactor: folder structure

kumaraswamy1 committed:
Update component.css

kumaraswamy1 committed:
refactor: folder structure

kumaraswamy1 committed:
refactor : Changing the id to eCommerce-card

kumaraswamy1 committed:
refactor: Changed path

kumaraswamy1 committed:
style : fixed extra padding for mobile devices

kumaraswamy1 committed:
style: Changed margin for h2

kumaraswamy1 committed:
style: updated card component

kumaraswamy1 committed:
Update card.css

kumaraswamy1 committed:
Update card.css

kumaraswamy1 committed:
style : updated inline margin.

Updated inline margin of e-commerce heading and fixed overflow by using ellipsis.

kumaraswamy1 committed:
style: updated eCommerce card margin

kumaraswamy1 committed:
style: Added remove button to eCommerce card

kumaraswamy1 committed:
style : updated class name

kumaraswamy1 committed:
style: fixed e-commerce badges

kumaraswamy1 committed:
style: fixed remove button positioning

kumaraswamy1 committed:
feat: updated toast component
